task_creator/cron_creator.py

```diff
@@ -0,0 +1,24 @@
+from scheduler.models.scheduled_task import CronTask, BaseTask
+from scheduler.models.args import TaskArg
+from django.contrib.contenttypes.models import ContentType
+
+
+def create_crontask(name, callable, cron_string, args=[], enabled=True, queue="evaluator"):
```

Comment:
```suggestion
def create_crontask(name: str, callable, cron_string: str, args=[], enabled=True, queue="evaluator"):
```
Could you please also add a documentation string that explains what this does?